### ISSUE TITLE
Missing package name in the generated code

### ISSUE THREAD

**Issue by fgrzadkowski** [ugorji/go#73]
My struct has field ```CreationTimestamp util.Time `json:"creationTimestamp,omitempty"```. Generated code uses just ```Time{}``` instead of ```util.Time{}```.

This is a regression introduced in one of the recent commits.

**Comment by ugorji** [comment]
Yes. Should have been qualified name. Will fix when I get back to desk.

**Comment by ugorji** [comment]
Regression from fix for #70

**Comment by fgrzadkowski** [comment]
This is still wrong, because now I get the following function names in generated code:
```
func (x codecSelfer3614) encSliceruntime.Object(v []runtime.Object, e *codec1978.Encoder) {
```
